packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -83,6 +83,16 @@
                 :is="errorBoundary"
                 tier="3">
                 <slot name="meta-items" />
+
+                <delivery-time-meta
+                    v-if="deliveryTimeData.eta || deliveryTimeData.distance || deliveryTimeData.address"
```

Comment:
could extract to a computed property named something like "displayDeliveryTimeMeta"